SharpNoise.Tests/ModuleTests/DiscreteTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SharpNoise.Modules;
+
+namespace SharpNoise.Tests.ModuleTests
+{
+    [TestClass]
+    public class DiscreteTests
+    {
+        Random random;
+
+        const Byte VALUES = 4;
+
+        public void DiscreteMainTestCode(Module source, Byte Count = VALUES)

```

Comment:
ok, but don't forget it's just a unit test.
Same as before, here I don't need so much for the tests, use a UInt16 instead, but it's again a waste of memory.